pkgs/development/tools/ddosify/default.nix

```diff
@@ -2,20 +2,22 @@
 
 buildGoModule rec {
   pname = "ddosify";
-  version = "0.8.0";
+  version = "0.8.1";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-ImVNiBXvKKYXuWtOajvLFobk956wNSQHLH7npdYY4SE=";
+    sha256 = "sha256-cedgJJd1+iWw3sxbKVBpi5XvmZdDcDL0sHhTELbkY9Q=";
   };
 
   vendorSha256 = "sha256-mq82KNa01gHvW+RUREra+ysaJ1YWIwX0v/uYMxmFN4M=";
 
   ldflags = [
     "-s -w"
     "-X main.GitVersion=${version}"
+    "-X main.GitCommit=${version}"
+    "-X main.BuildDate=1970-01-01T00:00:00"
```

Comment:
Doesn't quite work:
```
results/ddosify/bin/ddosify -version
Version:        0.8.1
Git commit:     0.8.1
Built           2022-07-25T04:46:25Z
Go version:     go1.18.4
OS/Arch:        darwin/amd64
```

Reply:
Seems that I got tricket, cheking it...

Reply:
Ah, I see why, all flags are a bit wrong because they're not split correctly. It should be
```
   "-s" "-w"
   "-X" "main..... 
```

Reply:
> Ah, I see why, all flags are a bit wrong because they're not split correctly. It should be
> 
> ```
>    "-s" "-w"
>    "-X" "main..... 
> ```

I should trust less in my sleepy version, but I still have problems with below diff:

```
diff --git a/pkgs/development/tools/ddosify/default.nix b/pkgs/development/tools/ddosify/default.nix
index e54bea0fc18..12d9f036a32 100644
--- a/pkgs/development/tools/ddosify/default.nix
+++ b/pkgs/development/tools/ddosify/default.nix
@@ -14,10 +14,10 @@ buildGoModule rec {
   vendorSha256 = "sha256-mq82KNa01gHvW+RUREra+ysaJ1YWIwX0v/uYMxmFN4M=";
 
   ldflags = [
-    "-s -w"
+    "-s" "-w"
     "-X main.GitVersion=${version}"
     "-X main.GitCommit=${version}"
-    "-X main.BuildDate=1970-01-01T00:00:00"
+    "-X main.BuildDate=1970-01-01T00:00:00Z"
   ];
 
   # TestCreateHammerMultipartPayload error occurred - Get "https://upload.wikimedia.org/wikipedia/commons/b/bd/Test.svg"
```

Reply:
Sorry, I deleted my comment, I was wrong, you don't actually have to split them :(

Reply:
No problem, will try to fix later, it seems that setting build date is being ignored, maybe a format issue?

Yet, 1970-01-01T00:00:00Z is a valid RFC3339 timestamp.

Reply:
So, little time to debug and not a good reason to leave this a stale, so I am doing the same as https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/cluster/argocd-autopilot/default.nix#L25.